<launch>
    <arg name="model_path" default="$(find lstm_land)/models/drone_policy_a6_best_v1.onnx"/>
    
    <!-- 决策节点 -->
    <node pkg="lstm_land" 
          type="ppo_node.py" 
          name="ppo_decision"
          output="screen"
          respawn="true">
        <!-- 私有参数传递 -->
        <param name="model_path" value="$(arg model_path)"/>
    </node>

    <!-- 观测节点 -->
    <include file="$(find lstm_land)/launch/observation.launch"/>
</launch>